packages/cloud_firestore/cloud_firestore_web/lib/src/interop/firestore.dart

```diff
@@ -110,8 +112,17 @@ class Firestore extends JsObjectWrapper<firestore_interop.FirestoreJsImpl> {
   }
 
   Future<Query> namedQuery(String name) async {
-    firestore_interop.QueryJsImpl query =
+    firestore_interop.QueryJsImpl? query =
         await handleThenable(jsObject.namedQuery(name));
+
+    if (query == null) {
+      // same error as iOS & android to maintain consistency
+      throw FirebaseException(
+          plugin: 'cloud_firestore',
+          message:
+              'Named query has not been found. Please check it has been loaded properly via loadBundle().');
```

Comment:
This needs an error code, so that people can differentiate this exception from others